qemu-user-blacklist.txt

```diff
@@ -29,13 +29,15 @@ fulcio
 gauche
 gawk
 gdk-pixbuf2
-glib-perl
 glibc
+glib-perl
```

Comment:
```python
In [1]: 'glibc' < 'glib-perl'
Out[1]: False

In [2]: ord('c'), ord('-')
Out[2]: (99, 45)
```

Reply:
```bash
╭─kxxt@spectre in repo: archriscv-packages on  patch-1 [!?] took 1ms
 ╰─λ LC_ALL=C sort qemu-user-blacklist.txt -o qemu-user-blacklist.txt

 ╭─kxxt@spectre in repo: archriscv-packages on  patch-1 [!?] took 4ms
 ╰─λ git status qemu-user-blacklist.txt 
On branch patch-1
Your branch is up to date with 'origin/patch-1'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   qemu-user-blacklist.txt

no changes added to commit (use "git add" and/or "git commit -a")

 ╭─kxxt@spectre in repo: archriscv-packages on  patch-1 [!?] took 6ms
 ╰─λ sort qemu-user-blacklist.txt -o qemu-user-blacklist.txt         

 ╭─kxxt@spectre in repo: archriscv-packages on  patch-1 [?] took 4ms
 ╰─λ git status qemu-user-blacklist.txt                     
On branch patch-1
Your branch is up to date with 'origin/patch-1'.

nothing to commit, working tree clean
```

My fault. I forget to change `LC_ALL` for `sort`. Should I make another PR to fix it or leave it untouched?

Reply:
> Should I make another PR to fix it or leave it untouched?

Better to have a fix. :)